python/lsst/ts/pmd/mock_server.py

```diff
@@ -97,3 +99,6 @@ def get_position(self, index):
             return f"{index}:{slot_position:+f}\r"
         else:
             return "\r"
+
+    def multiplexer_recovery(self):
```

Comment:
Consider a name that indicates this does nothing, e.g. do_nothing and add a comment about resetting the multiplexor where you assign the callback above. (If resetting had some logic then a specially named command handler or pair of handlers -- one each for the two required commands -- makes more sense. But in this case it does nothing.)

Please also consider prefixing command handlers with "do_" to make them more obvious., e.g. "do_nothing" and "do_get_position". This this particular mock server is so simple that's not very important, but it is always good practice. It tells the reader that these are all the same kind of method with the same basic "command handler" API.

Reply:
Please consider using the term "channel" instead of "slot" everywhere in this package, because that is typical naming for multiplexor and widely used in T&S. Unless the device's own documentation uses "slot"?